src/PlayerSummary.js

```diff
@@ -0,0 +1,54 @@
+import Affiliation from './Affiliations.js';
+import LoadOutInventory from './LoadOutInventory.js';
+import Mission from './Mission.js';
+import OperatorLoadOuts from './OperatorLoadOuts.js';
+import PlayerSkill from './PlayerSkill.js';
+import { parseDate } from './utils.js';
+
+export default class PlayerSummary {
+  constructor(summary) {
+    this.accountId = summary.AccountId.$oid;
+    this.displayName = summary.DisplayName;
+    this.playLevel = summary.PlayerLevel;
+    this.loadout = new LoadOutInventory(summary.LoadOutInventory);
+    this.playerSkills = new PlayerSkill(summary.PlayerSkills);
+
+    // TODO: See if the nightwave challenges use the same name in worldstate-data then translate
+    this.challengeProgress = summary.ChallengeProgress;
+    this.guildId = summary.GuildId.$oid;
+    this.guildName = summary.GuildName;
+    this.guildTier = summary.GuildTier;
+    this.guildXp = summary.GuildXp;
+    this.guildClass = summary.GuildClass;
+    this.guildEmblem = summary.GuildEmblem;
+    this.allianceId = summary.AllianceId.$oid;
+    this.deathMarks = summary.DeathMarks;
+    this.harvestable = summary.Harvestable;
+
+    // TODO: might be able to translate this once we update enemy data.
```

Comment:
Unfortunately the current enemy data in items can't really be updated... We could use this to make enemy data in worldstate-data

Reply:
I'll pull things that I have an add them to worldstate-data.